### ISSUE TITLE
fill discord id in sticker metadata request even if there is no metadata

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2810]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.42.8](https://github.com/open-wa/wa-automate-nodejs/compare/4.42.7...4.42.8) (2022-07-22)

- Release 4.42.8 [`59d239d`](https://github.com/open-wa/wa-automate-nodejs/commit/59d239deddd797b63f107b3769d63f4903039a55)
- fix: #2810 [`c5a06c6`](https://github.com/open-wa/wa-automate-nodejs/commit/c5a06c69dd3809ee7875adfc8defaf3c8f42f678)
- 🥚 patches update [`00f37e5`](https://github.com/open-wa/wa-automate-nodejs/commit/00f37e5361fadf5351d9a9ea496f40291c198e97)
- updated types-only package [`b509dcf`](https://github.com/open-wa/wa-automate-nodejs/commit/b509dcf199a0c6a4816fccbfee9111e1839bacaa)
